emuu-client/src/components/common/Button/Button.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
```

Comment:
Why not just implement this in the page we needed to? Don't think it's necessary to have a separate file for a button.